Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -101,12 +101,14 @@ class PreplannedMapModel: ObservableObject, Identifiable {
     private func updateStatus(for packagingStatus: PreplannedMapArea.PackagingStatus) {
         // Update area status for a given packaging status.
         switch packagingStatus {
-        case .processing:
-            status = .packaging
-        case .failed:
-            status = .packageFailure
-        case .complete:
-            status = .packaged
+            case .processing:
+                status = .packaging
+            case .failed:
+                status = .packageFailure
+            case .complete:
+                status = .packaged
+            @unknown default:
+                fatalError("Unknown case")
```

Comment:
The indentation is off here. Besides that, this is essentially type conversion (`PreplannedMapArea.PackagingStatus` -> `PreplannedMapModel.Status`), which is typically done in Swift via an initializer:

```swift
extension PreplannedMapModel.Status {
    init(packagingStatus: PreplannedMapArea.PackagingStatus) {
        self = switch packagingStatus {
        case .processing: .packaging
        case .failed: .packageFailure
        case .complete: .packaged
        @unknown default: fatalError("Unknown case")
        }
    }
}
```

Then, at the point of use, you can use it like so:

```swift
status = preplannedMapArea.packagingStatus.map(Status.init) ?? .packaged
```

That's what I suggest doing. A big reason that Swift prefers initializers for type conversion is that they can be used anywhere, whereas a method like `updateStatus(for:)` can only be used on this particular type.